Week3/BusinessCards/app/src/main/java/com/example/businesscards/MainActivity.kt

```diff
@@ -2,10 +2,74 @@ package com.example.businesscards
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+
 
 class MainActivity : AppCompatActivity() {
+    private val reminders = listOf("V = I x R", "I = V / R", "R = V / I", "P = V x I")
+
+    internal lateinit var currentReminder: String
```

Comment:
You don't need to use the `internal` modifier here I guess. If it's meant to be used just in this file, you can use private, otherwise, I'd rethink as to why this is not private!

Also try to avoid `lateinit var`s whenever possible.